Sources/SendGridKit/SendGridClient.swift

```diff
@@ -21,31 +21,34 @@ public struct SendGridClient: Sendable {
         return decoder
     }()
 
-    public init(httpClient: HTTPClient, apiKey: String) {
+    /// Initialize a new `SendGridClient`
+    ///
+    /// - Parameters:
+    ///   - httpClient: The `HTTPClient` to use for sending requests
+    ///   - apiKey: The SendGrid API key
+    ///   - forEU: Whether to use the API endpoint for global users and subusers or for EU regional subusers
+    public init(httpClient: HTTPClient, apiKey: String, forEU: Bool = false) {
         self.httpClient = httpClient
         self.apiKey = apiKey
+        self.apiURL = forEU ? "https://api.eu.sendgrid.com/v3/mail/send" : "https://api.sendgrid.com/v3/mail/send"
     }
     
     public func send(email: SendGridEmail) async throws {
         var headers = HTTPHeaders()
         headers.add(name: "Authorization", value: "Bearer \(apiKey)")
         headers.add(name: "Content-Type", value: "application/json")
+
+        var request = HTTPClientRequest(url: apiURL)
+        request.method = .POST
+        request.headers = headers
+        request.body = try HTTPClientRequest.Body.bytes(encoder.encode(email))
         
-        let response = try await httpClient.execute(
-            request: .init(
-                url: apiURL,
-                method: .POST,
-                headers: headers,
-                body: .data(encoder.encode(email))
-            )
-        ).get()
+        let response = try await httpClient.execute(request, timeout: .seconds(30))
         
         // If the request was accepted, simply return
-        guard response.status != .ok && response.status != .accepted else { return }
+        if response.status == .ok || response.status == .accepted { return }
```

Comment:
Better to check that status code is in range of 2xx.

Reply:
Like this?
```swift
if (200...299).contains(response.status.code) { return }
```

Reply:
Right